global.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
https://github.com/dotnet/sdk/issues/1421

That's my request for more documentation on this format. Hasn't gotten much of a response yet.